### PR TITLE
Minor Changes

### PR DESCRIPTION
**Pull request by qasimqlf**
No logic change, just selection names modified

### CODE REVIEW

**Review by phantinuss — APPROVED** [review]

**Comment by phantinuss** [comment]
@qasimqlf : If you open a PR and still plan on adding stuff to it, please mark the PR as draft. I didn't know you still wanted to add to it.

**Comment by qasimqlf** [comment]
> @qasimqlf : If you open a PR and still plan on adding stuff to it, please mark the PR as draft. I didn't know you still wanted to add to it.

Ok i forgot before. sorry. Now it's done

File: rules/windows/process_creation/proc_creation_win_apt_elise.yml [hunk 1]
```diff
@@ -17,12 +17,12 @@ logsource:
     category: process_creation
     product: windows
 detection:
-    selection1:
+    selection_cmd:
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_apt_elise.yml:20]:
```suggestion
    selection_1:
```

File: rules/windows/process_creation/proc_creation_win_apt_elise.yml [hunk 2]
```diff
@@ -17,12 +17,12 @@ logsource:
...
         Image: 'C:\Windows\SysWOW64\cmd.exe'
         CommandLine|contains: '\Windows\Caches\NavShExt.dll '
-    selection2:
+    selection_cli:
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_apt_elise.yml:23]:
```suggestion
    selection_2:
```

File: rules/windows/process_creation/proc_creation_win_apt_lazarus_activity_apr21.yml [hunk 3]
```diff
@@ -15,17 +15,17 @@ logsource:
     category: process_creation
     product: windows
 detection:
-    selection1:
+    selection_cli:
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_apt_lazarus_activity_apr21.yml:18]:
```suggestion
    selection_1:
```

File: rules/windows/process_creation/proc_creation_win_apt_lazarus_activity_apr21.yml [hunk 4]
```diff
@@ -15,17 +15,17 @@ logsource:
...
         CommandLine|contains|all:
             - 'mshta'
             - '.zip'
-    selection2:
+    selection_img_!:
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_apt_lazarus_activity_apr21.yml:22]:
```suggestion
    selection_2:
```

File: rules/windows/process_creation/proc_creation_win_apt_lazarus_activity_apr21.yml [hunk 5]
```diff
@@ -15,17 +15,17 @@ logsource:
...
         ParentImage: 'C:\Windows\System32\wbem\wmiprvse.exe'
         Image: 'C:\Windows\System32\mshta.exe'
-    selection3:
+    selection_img_2:
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_apt_lazarus_activity_apr21.yml:25]:
```suggestion
    selection_3:
```

**Review by nasbench — CHANGES_REQUESTED** [review]
@qasimqlf a couple of things.

- For ease of review please make your branch unprotected the next time you do a PR.
- There is no convention in SIGMA about the selection names it's an author preference :)

There is some logic behind it used in cases of ease of grouping the condition and ease of reading

File: rules/windows/process_creation/proc_creation_win_apt_winnti_pipemon.yml [hunk 6]
```diff
@@ -15,16 +15,16 @@ logsource:
...
+    selection_cmd_1:
         CommandLine|contains: 'setup.exe'
-    selection2b:
+    selection_cmd_2:
         CommandLine|endswith:
             - '-x:0'
             - '-x:1'
             - '-x:2'
-    condition: selection1 or all of selection2*
+    condition: selection_cli or all of selection_cmd_*
```
[the remark this answers is not in the document]
Comment (nasbench) [reply rules/windows/process_creation/proc_creation_win_apt_winnti_pipemon.yml:27]: You still didn't apply this.

**Review by nasbench — APPROVED** [review]

**Pull request merged**